src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeArm.java

```diff
@@ -0,0 +1,35 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import java.util.Collections;
+import java.util.LinkedList;
+
+public class NodeArm {
+    private int id;
+    private final LinkedList<NodeArm> neighbors;
```

Comment:
Any reason this should be `final`?

Reply:
when you set in once id doesnt change again